scalafix/rules/src/main/scala/fix/Cats_v1_0_0.scala

```diff
@@ -198,11 +128,13 @@ case class SimplifyEitherTLift(index: SemanticdbIndex)
 
   private[this] def removeWithLeadingComma(ctx: RuleCtx, t: Tree): Patch =
     (for {
-      leadingComma <- ctx.tokenList.leading(t.tokens.head).find(_.syntax == ",")
+      leadingComma <- ctx.tokenList
+        .leading(t.tokens.head)
+        .find(_.is[Token.Comma])
     } yield {
-      val leadingSpaces = ctx.tokenList.slice(leadingComma, t.tokens.head)
-      ctx.removeToken(leadingComma) +
-        leadingSpaces.map(ctx.removeToken).asPatch +
+      val leadingSpaces = ctx.tokenList.slice(leadingComma, t.tokens.last)
```

Comment:
are you sure about this change? The leading spaces are the once from the leading comma to the first token non-space token, what am I missing?

Reply:
whoops, that was a relic of a failed refactoring attempt.